tests/test_generics.py

```diff
@@ -530,8 +530,8 @@ def test_dynamic_serializer_form_in_browsable_api(self):
         view = DynamicSerializerView.as_view()
         request = factory.get('/')
         response = view(request).render()
-        self.assertContains(response, 'field_b')
-        self.assertNotContains(response, 'field_a')
+        assert response is 'field_b'
```

Comment:
This should be `field_b in response`. Right now it's doing an `is` check, which is definitely not what we were doing before.